Userland/Utilities/ini.cpp

```diff
@@ -31,9 +31,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
-    auto config = TRY(Core::ConfigFile::open(path, value_to_write ? Core::ConfigFile::AllowWriting::Yes : Core::ConfigFile::AllowWriting::No));
+    auto config = TRY(Core::ConfigFile::open(path, value_to_write.is_empty() ? Core::ConfigFile::AllowWriting::No : Core::ConfigFile::AllowWriting::Yes));
```

Comment:
Doesn't this break the case where you're setting some key to an empty string(`""`)?

Reply:
The config utility also uses is_empty(). May be this should be fixed.

Reply:
Yeah, definitely :)